code/modules/industrial_lift/industrial_lift.dm

```diff
@@ -58,7 +58,7 @@ GLOBAL_LIST_EMPTY(lifts)
 	/// if TRUE, we will gib anyone we land on top of. if FALSE, we will just apply damage with a serious wound penalty.
 	var/violent_landing = TRUE
 	/// damage multiplier if a mob is hit by the lift while it is moving horizontally
-	var/collision_lethality = 1
+	var/collision_lethality = 2
```

Comment:
shouldnt be used like this, this is meant to be a multiplier on the default damage of the tram. you should create defines for the lowest and largest base damage for the tram and increase those, then put them in /obj/structure/industrial_lift/proc/travel()

Reply:
We use collision_lethality during the tram malfunction event.

Reply:
But yeah I'll go edit base damage.